src/routes/RemediationDetails.js

```diff
@@ -47,6 +48,26 @@ class RemediationDetails extends Component {
         this.props.switchAutoReboot(this.id, autoReboot);
     };
 
+    handleSuccessBanner = (id, name) => {
```

Comment:
This function is not called anywhere, is that correct?

Reply:
At the moment, no it's not. But when the API is hooked up, this should be the function that is called when the playbook has finished.

Reply:
Nice planning ahead!